index.html

```diff
@@ -277,37 +277,83 @@ <h6 class="link-item">
                   </div>
                 </li>
                 <li class="code">
-                    <div class="code-template">
-                        <div class="code-img-b"></div>
-                        <p class="code-title">HTML</p>
-                      </div>
+                  <div class="code-template">
+                    <div class="code-img-b"></div>
+                    <p class="code-title">HTML</p>
+                  </div>
                 </li>
                 <li class="code">
-                    <div class="code-template">
-                        <div class="code-img-c"></div>
-                        <p class="code-title">CSS</p>
-                      </div>
+                  <div class="code-template">
+                    <div class="code-img-c"></div>
+                    <p class="code-title">CSS</p>
+                  </div>
                 </li>
               </ul>
             </div>
             <div class="skill-container">
-                <div class="skill-title">
-                  <ul>
-                    <li><a href="#" class="link-skill">Frameworks</a></li>
-                  </ul>
-                </div>
-                <div class="vector-icon"></div>
+              <div class="skill-title">
+                <ul>
+                  <li><a href="#" class="link-skill">Frameworks</a></li>
+                </ul>
               </div>
-              <div class="skill-container">
-                <div class="skill-title">
-                  <ul>
-                    <li><a href="#" class="link-skill">Skills</a></li>
-                  </ul>
-                </div>
-                <div class="vector-icon"></div>
+              <div class="vector-icon"></div>
+            </div>
+            <div class="skill-container">
+              <div class="skill-title">
+                <ul>
+                  <li><a href="#" class="link-skill">Skills</a></li>
+                </ul>
               </div>
+              <div class="vector-icon"></div>
+            </div>
           </div>
-          </div>
+          <section id="contact-section">
+            <div class="content contact-container">
+              <div class="contact-title-container">
+                <h2 class="contact-title">Contact me</h2>
+              </div>
+              <div class="contact-description-container">
+                <p class="contact-description">
+                  If you have an application you are interedte in developing, a
+                  feature that you need built or a project that need coding. I'd
+                  love to help with it
+                </p>
+              </div>
+              <div class="form-container">
+                <form class="contact-form">
+<<<<<<< HEAD
```

Comment:
- [ ] I think you have some merge issues in here, as you can see, this and some of the following lines are currently breaking your code:

![Screenshot_2021-10-29 Document](https://user-images.githubusercontent.com/66967274/139495819-164ebcf9-dc92-4bb2-af61-f7cd94a856ec.png)